front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -105,6 +105,12 @@ const SignUpInModal = memo<Props>(
       const authUserIsVerified =
         !isNilOrError(authUser) && authUser.attributes.verified;
 
+      // Temporary fix for CL-355 given urgency
+      if (metaData?.pathname.includes('projects/')) {
```

Comment:
Curious to know what this is about?

Reply:
This was part of another PR by Amanda, which was shown here because of my branch being rebased with a later version of master than the target branch. But I now changed the target branch back to master so it's gone :+1: